environments/example/vault.yml

```diff
@@ -12,3 +12,6 @@ vault_superset_fernet_key: fernet-key
 
 vault_oauth_client_id: client_id
 vault_oauth_client_secret: secret
+
+superset_admin_password: ""
```

Comment:
This comes from a [past PR](https://github.com/dimagi/commcare-analytics-ansible/pull/20/files) where I added this to the staging env but forgot to add it to the example env vault.